CHANGELOG.md

```diff
@@ -34,15 +34,16 @@
 
 ### Improvements
 
-- Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
+- **General:** Fix generation of metric names if any of ScaledObject's triggers is unavailable ([#2592](https://github.com/kedacore/keda/issues/2592))
```

Comment:
This entry if from v2.6.1. remove it from here please

Reply:
done